ios/NotificationService/NotificationService.swift

```diff
@@ -55,21 +55,22 @@ class NotificationService: UNNotificationServiceExtension {
     if #available(iOSApplicationExtension 15.0, *) {
       let overrideUsername = notification.userInfo["override_username"] as? String
       let senderId = notification.userInfo["sender_id"] as? String
-      let sender = overrideUsername ?? senderId
```

Comment:
setting the senderId as the override username caused a crash trying to construct an URL when the override username is using non latin words.